configure

```diff
@@ -257,6 +257,7 @@ External library support:
                            for DNN based filters like dnn_processing [no]
   --enable-libopus         enable Opus de/encoding via libopus [no]
   --enable-libplacebo      enable libplacebo library [no]
+  --enable-libproresdecoder enable Apple ProRes decoder library [no]
```

Comment:
This is the enable / disable option for FFmpeg compliation. We haven't set it for a default enable